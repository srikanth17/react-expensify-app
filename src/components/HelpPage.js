import React from 'react';

const HelpPage = () => (
    <div>
        This is from helppage component
    </div>
);

export default HelpPage;