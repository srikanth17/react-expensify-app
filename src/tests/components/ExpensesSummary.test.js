import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should correctly render expense summary with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render expense summary with multiple expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={13} expensesTotal={235534566}/>);
    expect(wrapper).toMatchSnapshot();
});