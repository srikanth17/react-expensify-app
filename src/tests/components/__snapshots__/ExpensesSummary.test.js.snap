// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expense summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $235.00
  </h1>
</div>
`;

exports[`should correctly render expense summary with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    13
     
    expenses
     totalling 
    $235,534,566.00
  </h1>
</div>
`;
